tests/builder/axes_test.py

```diff
@@ -342,6 +342,145 @@ def test_mapping_using_axis_location_custom_parameter_on_instances(ufo_module):
     assert doc.axes[1].map == [(50, 224), (60, 384), (100, 448)]
 
 
+def test_mapping_using_axis_location_cp_on_masters_none(ufo_module):
+    # https://github.com/googlefonts/glyphsLib/issues/714
+    # https://github.com/googlefonts/glyphsLib/pull/810
+
+    # If custom parameters are disabled, they return None.
+    # The master Axis Location CP are disabled, but the ones on the instances
```

Comment:
I'm curious, how common would be to have this Axis Location custom parameter in a master _and_ have it marked as "disabled"... Why would one want to do that? Wouldn't it be sufficient to not have it there/delete it?
I ask because you added two tests that specifically test for that None-ness

Reply:
I set them to "disabled" all the time, to compare versions or test something. But I wasn't aware that it always returns None, no matter if it is disabled or not there at all. Will adapt the code.